spec/features/home_spec.rb

```diff
@@ -3,9 +3,12 @@
 describe "main page tests" do
   let(:user) { FactoryGirl.create :user, email: "unique@example.com",
                                          password: "password" }
+  let(:deck) { FactoryGirl.create :deck, title: "title" }
   let(:second_user) { FactoryGirl.create :user }
   let(:card) { FactoryGirl.create :card, translated_text: " Correct value ",
-                                         user_id: user.id }
+                                         user_id: user.id,
+                                         deck_id: deck_id 
+             }
```

Comment:
`end` at 11, 13 is not aligned with `let(:card) { FactoryGirl.create :card, translated_text: " Correct value ",` at 8, 2